settings/dev.sample.py

```diff
@@ -8,11 +8,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'evalai',
-        'USER': 'postgres',
-        'PASSWORD': 'postgres',
-        'HOST': 'localhost',
-        'PORT': 5432,
+        'NAME': os.environ.get("POSTGRES_NAME", 'evalai'),  # noqa: ignore=F405
```

Comment:
Flake8 is complaining about the `os` module. Since we are importing it from `common.py` it felt easier to just ignore the error for those lines.